files/image_config/interfaces/interfaces.j2

```diff
@@ -18,6 +18,10 @@ iface lo {{ 'inet' if prefix | ipv4 else 'inet6' }} static
 {% endfor %}
 {% endblock loopback %}
 {% block mgmt_interface %}
+# BMC interface
+auto usb0
+iface usb0 inet6 auto
```

Comment:
what if there is no usb0?

Reply:
If I understand correctly, I could remove this from here in the template file. But the output generated using cfggen with this file is going to be compared with a sample interfaces file which is fixed. If I keep the usb0 instance there and not here (conditionally), then sonic build shall itself fail.

Reply:
then you can remove the usb0 in the cfggen sample interface file.

Reply:
fixed